### ISSUE TITLE
Cannot Skip User’s Playback To Previous Track - 403: "Command failed: Restriction violated"

### ISSUE THREAD

**Issue by ashthespy** [spotify/web-api#1038]
#### Cannot Skip User’s Playback To Previous Track 

Issue found on September 22nd 2018.

#### Endpoint(s):
* `POST https://api.spotify.com/v1/me/player/previous`

#### Scope(s):
* user-modify-playback-state

#### Steps to reproduce:
1. Open the API Console
2. Start playback with `PUT | /v1/me/player/play`
3. Skip to next track with `POST | /v1/me/player/next`

4. Skip to previous track with `POST | /v1/me/player/previous`

#### Expected behaviour:
Device skips to previous track.   

#### Actual behaviour:
Command failed: Restriction violated
```
{
  "error": {
    "status": 403,
    "message": "Command failed: Restriction violated"
  }
}
```

**Comment by ashthespy** [comment]
@arirawr this was with Spotify Premium.

**Comment by ashthespy** [comment]
@arirawr I haven't tried of late - I had the issue multiple times back then when I opened up the issue, back in September. 

If it also is of significance, this issue was seen also on a Spotify Connect endpoint (the webplayer for example) 

Either way, shall update the issue if I stumble onto it again!

**Comment by arirawr** [comment]
Alright, I'll go ahead and close this then but please reopen with reproduction steps if you see it again.

**Comment by SidMurthy97** [comment]
Hi I got this issue when using the start_playback() function in python. The issue was because I was calling the start_playback() function when I was already playing a song, so the API crashed. Not sure what the equivalent would be in your case, but I solved it by adding some padding in my case

**Comment by jacksonmelcher** [comment]
I also have this issue only when calling the API within javascript. When I use postman it works.

**Comment by rws2012** [comment]
I figured out ONE reason why this happens (this may not be the only reason), and that is because I tried to go back a song when there was no song to go back to. I only just now figured this out when I tried going to the previous song on the actual spotify application itself and wasn't able to. Hopefully this helps someone else out because I've been trying to figure out why I was having this problem for like 2 hours.
